README.md

```diff
@@ -304,6 +304,24 @@ The following settings can be optionally set to customize the node labels and ta
 The following settings are optional and allow you to further configure your cluster.
 * `settings.kubernetes.cluster-domain`: The DNS domain for this cluster, allowing all Kubernetes-run containers to search this domain before the host's search domains.  Defaults to `cluster.local`.
 
+You can also optionally specify static pods for your node with the following settings.
+* `settings.kubernetes.static-pods.<custom identifier>.manifest`: Base64-blob of the pod's manifest.
+* `settings.kubernetes.static-pods.<custom identifier>.enabled`: Whether the static pod is enabled or not.
+
+Here's an example of adding a nginx static pod with API calls:
+```
+apiclient set kubernetes.static-pods.nginx.manifest=YXBpVmVyc2lvbjogdjEKa2luZDogUG9kCm1ldGFkYXRhOgogICBuYW1lOiBzdGF0aWMtcG9kLW5naW54LXRlc3QKc3BlYzoKICAgaG9zdE5ldHdvcms6IHRydWUKICAgY29udGFpbmVyczoKICAgICAgLSBuYW1lOiBuZ2lueAogICAgICAgIGltYWdlOiBuZ2lueDoxLjE0LjIKICAgICAgICBwb3J0czoKICAgICAgICAgICAtIGNvbnRhaW5lclBvcnQ6IDgwCg==
+apiclient set kubernetes.static-pods.nginx.enabled=true
+```
+
+Here's the same example, but with the settings you'd add to user data:
+```
+[settings.kubernetes.static-pods.nginx]
+enabled = true
+manifest = "YXBpVmVyc2lvbjogdjEKa2luZDogUG9kCm1ldGFkYXRhOgogICBuYW1lOiBzdGF0aWMtcG9kLW5naW54LXRlc3QKc3BlYzoKICAgaG9zdE5ldHdvcms6IHRydWUKICAgY29udGFpbmVyczoKICAgICAgLSBuYW1lOiBuZ2lueAogICAgICAgIGltYWdlOiBuZ2lueDoxLjE0LjIKICAgICAgICBwb3J0czoKICAgICAgICAgICAtIGNvbnRhaW5lclBvcnQ6IDgwCg=="
+```
```

Comment:
I'd prefer not to include examples for this; the very long lines of base64 are somewhat jarring on in our main README.

It's also a bit weird showing both user-data and `apiclient` examples, since we don't do that for other settings.